About Fixed Position

KyumKyum opened the pull request:
fixed position

Commit:
test

Commit:
announcement

Commit:
changed

Commit:
edited by KimTaeHyeon

SeongIkKim committed:
[CREATE] ProfileCell skeletion

SeongIkKim committed:
[CREATE] first semester skeleton

SeongIkKim committed:
[CREATE] basic components in staffs

SeongIkKim committed:
[CREATE] json load by sideList

SeongIkKim committed:
[CREATE] Component directory settings & dummy file

SeongIkKim committed:
Merge branch 'dev' of https://github.com/hforif/forif-site into feature_staffs

SeongIkKim committed:
[FIX] Component layout & import directions

SeongIkKim committed:
Merge branch 'dev' of https://github.com/hforif/forif-site into feature_staffs

SeongIkKim committed:
[FIX] layout import path in staffs.js

SeongIkKim committed:
[FIX] staffs path

SeongIkKim committed:
[CREATE] Hackaton skeleton

SeongIkKim committed:
[CREATE] HackOverview without modal

Commit:
About Feature

JungeunK-9999 committed:
dev브랜치에서 git merge feature_About

SeongIkKim committed:
[CREATE] Hackathon modal & settings for graphql

JungeunK-9999 committed:
About.js -> index.js 옮기는 과정에서 오류 수정

Commit:
Carousel

Commit:
text changed (right aligned)

Commit:
history section edited

KyumKyum committed:
Merge pull request #4 from hforif/feature_About

Feature about  About - Carousel

YerinCho commented on src/pages/Staffs.js:
```diff
@@ -0,0 +1,22 @@
+export default function Staffs({location}) {
+  const [shown, setShown] = useState(0);
+  
+  let yearList = [];
+  const currentYear = new Date().getFullYear();
+  for (var i=2017; i<=currentYear; i++){
+      yearList.unshift(i);
+  }
+  const year = yearList[shown];
```
```suggestion
  const currentYear = new Date().getFullYear();
  const yearList = Array.from({length: currentYear - 2016}, (x, i) => currentYear - i);
  const year = yearList[shown]; 
```

YerinCho commented on src/pages/index.js:
```diff
@@ -1,11 +1,104 @@
   return (
-    <Layout pathName={location.pathname}>
-      <h1>This is your Forif Home!</h1>
+    <Layout sideList={0} pathName={location.pathname}>
```
컴포넌트 분리

YerinCho commented on src/components/staffs/Profile.js:
```diff
@@ -0,0 +1,58 @@
+import styles from "./Profile.module.css"
+// import profileInfo from "./profileInfo.json"
+
+function Profile ({year}) {
```
```suggestion
const Profile = () => ({year}) {
```
하나로 통일해요~

YerinCho commented on src/components/staffs/Profile.js:
```diff
@@ -0,0 +1,58 @@
+import React from "react"
```
여기 Profile, SemesterBlock, ProfileCell 파일 세개로 나눕시다

YerinCho reviewed (commented):
react와 css가 익숙해지면
[styled component](https://styled-components.com/)

Commit:
studies page and mobile supported

JungeunK-9999 committed:
포리프 인스타그램 피드 홈 화면에 붙임

JungeunK-9999 committed:
instagram feed 컴포넌트 분리. 그리고 약간의 css

JungeunK-9999 committed:
환경변수 설정

set env for netlify-deploy
prefix GATSBY_
env testing
netlify Environment variables set

SeongIkKim committed:
Merge branch 'feature/hackathon' into dev

Commit:
layout edited and studies featured

Commit:
font added

kimtahen committed:
Merge pull request #6 from hforif/feature/instagram_feed

Feature/instagram_feed

Commit:
edit layout & announcement & studies

Commit:
after merge with dev & layout edited

JungeunK-9999 committed:
Merge pull request #7 from hforif/feature_studies

dev merge 후 레이아웃을 수정했습니다.
